src/animators/baseAnimator.ts

```diff
@@ -37,8 +36,8 @@ export module Animators {
      */
     public static DEFAULT_EASING = "exp-out";
 
-    private _duration: number;
     private _delay: number;
+    private _duration: number;
```

Comment:
why do these still have _? It is confusing to read. I believe you should fix these in this PR as well